### DISCUSSION TITLE
ComboBox Background change on pointerover

### DISCUSSION THREAD

**Discussion by jackydebu** [AvaloniaUI/Avalonia#13034] [Q&A]
Hi everyone, I'm trying to change the hover color of my combobox but it doesn't seem to work, following the documentation. I can't figure out what I'm doing wrong, any suggestions?

What I've tried:
`<Window.Styles>
	<Style Selector="ComboBox">
		<Setter Property="BorderThickness" Value="0"/>
		<Setter Property="Background" Value="#A94DC1"/>
	</Style>
	<Style Selector="ComboBox:pointerover">
		<Setter Property="Background" Value="Red"/>
	</Style>
</Window.Styles>`

My combo box:
`<ComboBox Grid.Column="1" Name="ToolsComboBox" Width="128" Height="28"  Margin="4,5,4,0">
	<ComboBox.Styles>
		<Style Selector="ComboBox:pointerover">
			<Setter Property="Background" Value="Red"/>
		</Style>
	</ComboBox.Styles>
</ComboBox>`

**Comment by workgroupengineering** [comment]
see [this](https://docs.avaloniaui.net/docs/styling/troubleshooting#selector-with-a-pseudoclass-doesnt-override-the-default).

**Comment by jackydebu** [comment]
`<ComboBox Grid.Column="1" Name="ToolsComboBox" Width="128" Height="28"  Margin="4,5,4,0">
	<ComboBox.Styles>
		<Style Selector="ComboBox:pointerover /template/ Border#Background">
			<Setter Property="Background" Value="Red"/>
		</Style>
	</ComboBox.Styles>
</ComboBox>`

This works, thank you kind stranger!

**Reply by timunie** [reply]
@jackydebu always worth to look up original styles in source. This often helps me to understand what needs to change.
